### ISSUE TITLE
Is there a function to check CPU/memory usage?

### ISSUE THREAD

**Issue by DragonClub** [danielpaulus/go-ios#433]
I think it would be quite nice if there was one.

**Comment by hazmi-e205** [comment]
I agree for this feature, I planned to use go-ios instead of gidevice as the backend of this tool https://github.com/SonicCloudOrg/sonic-ios-bridge/ but seems some features missing especially

- Perf Monitor (https://github.com/SonicCloudOrg/sonic-ios-bridge/blob/main/cmd/perfmon.go)
- Share device via network (https://github.com/SonicCloudOrg/sonic-ios-bridge/tree/main/cmd/remote)

**Comment by hazmi-e205** [comment]
Someone already working on this feature
https://github.com/danielpaulus/go-ios/pull/475

**Comment by danielpaulus** [comment]
Closing during issue triage: current go-ios appears to include this behavior already, or the affected area has been substantially replaced since this was opened. If the problem still exists on the latest release, please open a fresh issue or ask us to reopen this one with current command output and version details.
